programs/programs/mo/pe/spm.py

```
@@ -0,0 +1,33 @@
+
+        state_code=MO   -> school_meal_net_subsidy $0.00      school_meal_tier PAID
+        state_code omitted -> school_meal_net_subsidy $1130.96 school_meal_tier FREE
+
+    So without this input every Missouri household would be shown eligible for
+    free meals regardless of income. ``test_spm.py`` pins it for that reason.
+    """
+
+    pe_inputs = [
+        *SchoolLunch.pe_inputs,
+        dependency.household.MoStateCodeDependency,
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)